bench/Speed/Tidal/PatternB.hs

```diff
@@ -1,44 +1,15 @@
 module Tidal.PatternB where 
 
-import Criterion.Main
-import Tidal.Inputs
-import Sound.Tidal.Pattern
+import           Criterion.Main    (Benchmark, bench, whnf)
+import           Sound.Tidal.Arc   (hull, sect)
+import           Sound.Tidal.Types (ArcF (Arc))
 
-arc1 = Arc 3 5 
+arc1 = Arc 3 5
 arc2 = Arc 4 6
 arc3 = Arc 0 1
 arc4 = Arc 1 2
 
-withQueryTimeB :: [Benchmark] 
-withQueryTimeB = 
-  [ bgroup "withQueryTime" [
-      bench "wqt whnf" $ whnf withQueryTime (*2) 
-    , bench "wqt2 whnf" $ whnf withQueryTime (+1)
-    , bench "wqt nf" $ nf withQueryTime (*2) ]
-  ]
-
-withResultArcB :: [Benchmark]
```

Comment:
This is now [withEventArc](https://github.com/tidalcycles/Tidal/blob/78695cd30137fabaad0164ac490ba8c0c26d6bc4/src/Sound/Tidal/Signal/Base.hs#L373-L377)